From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Tue, 27 Jun 2023 13:49:54 +0200
Subject: Cargo.*: use xkbcommon v0.5 for "newer" builds

Debian now provides `rust-xkbcommon` v0.5, so let's use this as the
base version for our use-case.
---
 Cargo.deps       | 3 ++-
 Cargo.deps.newer | 1 +
 Cargo.toml.in    | 1 -
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/Cargo.deps b/Cargo.deps
index e7c5e06..079341e 100644
--- a/Cargo.deps
+++ b/Cargo.deps
@@ -5,6 +5,7 @@ zbus = "1.0.*"
 zvariant = "2.0.*"
 # Newer versions seem to confuse the version of Cargo on Debian Bullseye
 zvariant_derive = "2.0.*"
+xkbcommon = { version = "0.4.*", features = ["wayland"] }
 
 [dependencies.cairo-rs]
 version = "0.7.*"
diff --git a/Cargo.deps.newer b/Cargo.deps.newer
index a6d4eac..57baa23 100644
--- a/Cargo.deps.newer
+++ b/Cargo.deps.newer
@@ -7,6 +7,7 @@ zbus = "1.9.*"
 zvariant = "2.10.*"
 # Newer versions seem to confuse the version of Cargo on Debian Bullseye
 zvariant_derive = "2.10.*"
+xkbcommon = { version = "0.5.*", features = ["wayland"] }
 
 [dependencies.cairo-rs]
 version = "0.16.*"
diff --git a/Cargo.toml.in b/Cargo.toml.in
index c9b2064..557e6c6 100644
--- a/Cargo.toml.in
+++ b/Cargo.toml.in
@@ -31,5 +31,4 @@ clap_v4 = []
 maplit = "1.0.*"
 serde = { version = "1.0.*", features = ["derive"] }
 serde_yaml = "0.8.*"
-xkbcommon = { version = "0.4.*", features = ["wayland"] }
 # Here is inserted the Cargo.deps file
